### DISCUSSION TITLE
Multiple images shows in results when using GAN in txt2img

### DISCUSSION THREAD

**Discussion by DasWookie** [hlky/stable-diffusion-webui#901] [General]
How in the world can I make the results window only show the final result when using either of the two GAN options? when rending a large number of images, the pre-gan results are useless as they aren't saved, and all it does it crowd the results window. Plus, it's taking up extra memory which I admit isn't a HUGE issue, but still, it's unnecessary, or at the very least should be optional to show the pre-gan result with the default being off.

![image](https://user-images.githubusercontent.com/9283270/189371811-9e5d4c3b-dac9-4fab-8f84-f3fb10251ae7.png)

(This happens with fix faces or upscale)

**Comment by dudumaroja** [comment]
i like it. it give you a option to push the modifed face to img2img in case you didnt liked the gan result

**Comment by Heathen** [comment]
I think a bigger issue is it being pushed as a data instead of an image link. In a limited environment, like for example using a weak cellphone browser, it quickly slogs down to a halt when it displays several images.
